languages/translations/en.json

```diff
@@ -238,37 +238,51 @@
             "default": {
                 "default": "Take the ramp",
                 "name": "Take the ramp onto {way_name}",
-                "destination": "Take the ramp towards {destination}"
+                "destination": "Take the ramp towards {destination}",
+                "exit": "Take exit {exit}",
+                "exit_destination": "Take exit {exit} towards {destination}"
             },
             "left": {
                 "default": "Take the ramp on the left",
                 "name": "Take the ramp on the left onto {way_name}",
-                "destination": "Take the ramp on the left towards {destination}"
+                "destination": "Take the ramp on the left towards {destination}",
+                "exit": "Take exit {exit} on the left",
```

Comment:
@freenerd just saw your note further down:

> Decide on exit directions (i vote for remove left/right)

I learn toward more specificity here because it can be super jarring when there is a left exit and driver really do expect it to be specifically called out.